Problems installing on windows 8.1 x64 - Python 2.7.8

I have tried all pip, conda and git options but looks like some header or library is missing:
###### _lrucache.obj : error LNK2019: unresolved external symbol _PyGC_generation0 referenced in function make_key

--- a/src/_lrucache.c
+++ b/src/_lrucache.c
@@ -636,7 +636,7 @@ make_key(cacheobject *co, PyObject *args, PyObject *kw)
 
   Py_SIZE(lo) = size;
   lo->allocated = size;
-  _PyObject_GC_TRACK(hs);
+  PyObject_GC_Track(hs);
   // incorporate extra state
   if(is_list){
     for(i = 0; i < ex_size; i++){

